web/src/components/datasets/DatasetColumnLineage.tsx

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import { Box } from '@material-ui/core'
+import MqEmpty from '../core/empty/MqEmpty'
+import MqJson from '../core/code/MqJson'
+import React, { FunctionComponent } from 'react'
+
+interface DatasetColumnLineageProps {
+  columnLineage: object
+}
+
+const DatasetColumnLineage: FunctionComponent<DatasetColumnLineageProps> = props => {
+  const { columnLineage } = props
+
+  return (
+    <Box>
+      {columnLineage
+        ? (
+          <Box mt={2}>
+            <MqJson code={columnLineage} />
+          </Box>
```

Comment:
Did we end up incorporating a size restriction to rendering JSON. I think this could potentially overwhelm a browser if it's several megs like some other facet payloads.